support-frontend/app/services/stepfunctions/StateWrapper.scala

```diff
@@ -21,8 +20,10 @@ class StateWrapper(encryption: EncryptionProvider, useEncryption: Boolean) {
   implicit private val wrapperEncoder = deriveEncoder[JsonWrapper]
   implicit private val wrapperDecoder = deriveDecoder[JsonWrapper]
 
+  val utf8 = "UTF-8"
```

Comment:
I think the easiest way to address the rest of my feedback is to replace this with: 

```scala
val utf8 = java.nio.charset.StandardCharsets.UTF_8
```